src/Theme/MarkOneWrapper.tsx

```diff
@@ -1,10 +1,26 @@
 import React, {
-  ReactElement, ReactNode, FunctionComponent,
+  ReactElement,
+  ReactNode,
+  FunctionComponent,
+  Ref,
 } from 'react';
 import { ThemeProvider } from 'styled-components';
 import GlobalCSS from './GlobalCSS';
 import MarkOneTheme from './MarkOneTheme';
 
+/**
+ * This interface includes props that are common to many components for testing
+ * and accessibility purposes.
+ */
+export interface MarkOneProps<T extends HTMLElement> {
+  /** Specifies the ref of the element */
+  forwardRef?: Ref<T>;
+  /** Corresponds to HTML attribute tabindex */
+  tabIndex?: number;
```

Comment:
I did not understand the usage of the tabIndex here

Reply:
Some elements cannot be "tabbed" to using the tab key, but I just realized that buttons are "tab-able," so I removed the tab index from the button elements (1988d6d). However, I believe there will be some elements in the future are not included in this list of [elements that can receive focus](https://stackoverflow.com/questions/1599660/which-html-elements-can-receive-focus), so I think it makes sense to keep `tabIndex` as a property in the interface.
This is the doc I referenced for the [tabindex attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex).

Reply:
Ok make sense, thanks for the explanation and links.